test/views/item/AddAnotherItemViewSpec.scala

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package views
+package views.item
```

Comment:
Accidental diff? Doesn't seem to correlate with the location of the view its testing